### PR TITLE
.github:refine issue template

### PR DESCRIPTION
**Pull request by AndreMouche**
###  What have you changed?

Refine the issue template

###  What is the type of the changes?
- Engineering (engineering change which doesn't change any feature or fix any issue)

###  How is the PR tested?
- No code

###  Does this PR affect documentation (docs) or should it be mentioned in the release notes?
no 

###  Does this PR affect `tidb-ansible`?
no

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/development-task.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+name: "\U0001F680 Development Task"
+about: As a TiDB developer, I want to record a development task.
```
Comment (BusyJay) [inline .github/ISSUE_TEMPLATE/development-task.md]: You mean `TiKV`?

File: .github/ISSUE_TEMPLATE/development-task.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+---
+name: "\U0001F680 Development Task"
```
Comment (BusyJay) [inline .github/ISSUE_TEMPLATE/development-task.md]: Isn't is should be one of bug/feature-request?

File: .github/pull_request_template.md [hunk 3]
```diff
@@ -4,51 +4,46 @@ Thank you for contributing to TiKV!
 If you haven't already, please read TiKV's [CONTRIBUTING](https://github.com/tikv/tikv/blob/master/CONTRIBUTING.md) document.
 
 If you're unsure about anything, just ask; somebody should be along to answer within a day or two.
+
+PR Title Format:
+1. pkg [, pkg2, pkg3]: what's changed
```
Comment (BusyJay) [inline .github/pull_request_template.md]: s/pkg/module/

File: .github/pull_request_template.md [hunk 4]
```diff
@@ -4,51 +4,46 @@ Thank you for contributing to TiKV!
...
-Pick one of the following and delete the others:
--->
-- New feature (a change which adds functionality)
-- Improvement (a change which is an improvement to an existing feature)
-- Bugfix (a change which fixes an issue)
-- Engineering (engineering change which doesn't change any feature or fix any issue)
-- Misc (other changes)
+Proposal: [xxx](url) <!-- REMOVE this line if not applicable -->
+
+What's Changed:
```
Comment (BusyJay) [inline .github/pull_request_template.md]: Usually these two share common content and hard to be separated.

**Comment by BusyJay** [comment]
@you06 Why merge this? Can you answer the review comments first?

File: .github/pull_request_template.md (hunk 4 above)
Comment (you06) [reply .github/pull_request_template.md]: Do you mean "What's Changed" and "How it Works" should be combined into one section?

File: .github/ISSUE_TEMPLATE/development-task.md (hunk 2 above)
Comment (you06) [reply .github/ISSUE_TEMPLATE/development-task.md]: IMO, bug typed issues should use bug report template.

File: .github/pull_request_template.md (hunk 4 above)
Comment (BusyJay) [reply .github/pull_request_template.md]: Yes. According to my experience, they are hardly separated.

File: .github/ISSUE_TEMPLATE/development-task.md (hunk 2 above)
Comment (BusyJay) [reply .github/ISSUE_TEMPLATE/development-task.md]: What I mean is development task either solves a bug or implements a new feature.

**Pull request merged**